hvplot/tests/testgeowithoutgv.py

```diff
@@ -83,3 +86,20 @@ def test_plot_with_dask(self, simple_df):
         assert isinstance(plot.get(0), hv.Tiles)
         bk_plot = bk_renderer.get_plot(plot)
         assert bk_plot.projection == 'mercator'
+
+    @pytest.mark.skipif(spd is None, reason='spatialpandas not installed')
+    def test_plot_without_crs(self):
+        square = spd.geometry.Polygon([(0.0, 0), (0, 1), (1, 1), (1, 0)])
+        sdf = spd.GeoDataFrame({'geometry': spd.GeoSeries([square, square]), 'name': ['A', 'B']})
+        plot = sdf.hvplot.polygons(tiles=True)
+
+        if hasattr(sdf, 'crs'):
+            del sdf.crs
+
+        assert len(plot) == 2
+        assert is_geodataframe(sdf)
+        assert not hasattr(plot, 'crs')
+        assert isinstance(plot.get(0), hv.Tiles)
+        assert isinstance(plot.get(1), hv.Polygons)
+        bk_plot = bk_renderer.get_plot(plot)
+        assert bk_plot.projection == 'mercator'
```

Comment:
Should there be a projection if the crs has been deleted or is this just the default?

Reply:
OK!